sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/BeamBuiltinAggregations.java

```diff
@@ -426,11 +515,55 @@ public Long mergeAccumulators(Iterable<Long> accums) {
     }
 
     @Override
-    public Long extractOutput(Long accum) {
+    public Long extractOutput(Long accumulator) {
       if (this.isEmpty) {
         return null;
       }
-      return accum;
+      return accumulator;
+    }
+  }
+
+  static CombineFn createLogicalAnd(Schema.FieldType fieldType) {
+    if (fieldType.getTypeName() == TypeName.BOOLEAN) {
+      return new LogicalAnd();
+    }
+    throw new UnsupportedOperationException(
+        String.format("[%s] is not supported in LOGICAL_AND", fieldType));
```

Comment:
Nit: add to this message, "LOGICAL_AND only supports booleans."

Reply:
done.